spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-kubernetes.adoc

```diff
@@ -239,32 +237,38 @@ To control what version of Skipper gets deployed, modify the tag used for the Do
 +
-If you want to orchestrate stream processing pipelines with Apache Kafka as the messaging middleware by using Skipper, you must change
-the `SPRING_APPLICATION_JSON` environment variable value in the `src/kubernetes/skipper/skipper-deployment.yaml` file as follows:
+If you would like to use RabbitMQ as the messaging middleware, apply the following:
 +
 ====
-[source,yaml]
+[source,bash]
 ----
-"{\"spring.cloud.skipper.server.enableLocalPlatform\" : false, \"spring.cloud.skipper.server.platform.kubernetes.accounts.minikube.environmentVariables\": \"SPRING_CLOUD_STREAM_KAFKA_BINDER_BROKERS=${KAFKA_SERVICE_HOST}:${KAFKA_SERVICE_PORT}, SPRING_CLOUD_STREAM_KAFKA_BINDER_ZK_NODES=${KAFKA_ZK_SERVICE_HOST}:${KAFKA_ZK_SERVICE_PORT}\",\"spring.cloud.skipper.server.platform.kubernetes.accounts.minikube.memory\" : \"1024Mi\"}"
+kubectl create -f src/kubernetes/skipper/skipper-config-rabbit.yaml
 ----
 ====
 +
-Additionally, if you want to use the link:https://docs.spring.io/spring-cloud-stream/docs/current/reference/htmlsingle/#_apache_kafka_streams_binder[Apache Kafka Streams Binder], configure the `SPRING_APPLICATION_JSON` environment variable in `src/kubernetes/skipper/skipper-deployment.yaml` as follows:
+If you would like to use Apache Kafka as the messaging middleware, apply the following:
++
+====
+[source,bash]
+----
+kubectl create -f src/kubernetes/skipper/skipper-config-kafka.yaml
+----
+====
++
+Additionally, if you want to use the link:https://docs.spring.io/spring-cloud-stream/docs/current/reference/htmlsingle/#_apache_kafka_streams_binder[Apache Kafka Streams Binder], update the `environmentVariables` attribute to include the Kafka Streams Binder configuraton in `src/kubernetes/skipper/skipper-config-kafka.yaml` as follows:
 +
 ====
 [source,yaml]
 ----
-"{\"spring.cloud.skipper.server.enableLocalPlatform\" : false, \"spring.cloud.skipper.server.platform.kubernetes.accounts.minikube.environmentVariables\": \"SPRING_CLOUD_STREAM_KAFKA_BINDER_BROKERS=${KAFKA_SERVICE_HOST}:${KAFKA_SERVICE_PORT}, SPRING_CLOUD_STREAM_KAFKA_BINDER_ZK_NODES=${KAFKA_ZK_SERVICE_HOST}:${KAFKA_ZK_SERVICE_PORT}, SPRING_CLOUD_STREAM_KAFKA_STREAMS_BINDER_BROKERS=${KAFKA_SERVICE_HOST}:${KAFKA_SERVICE_PORT}, SPRING_CLOUD_STREAM_KAFKA_STREAMS_BINDER_ZK_NODES=${KAFKA_ZK_SERVICE_HOST}:${KAFKA_ZK_SERVICE_PORT}\",\"spring.cloud.skipper.server.platform.kubernetes.accounts.minikube.memory\" : \"1024Mi\"}"
+environmentVariables: 'SPRING_CLOUD_STREAM_KAFKA_BINDER_BROKERS=${KAFKA_SERVICE_HOST}:${KAFKA_SERVICE_PORT},SPRING_CLOUD_STREAM_KAFKA_BINDER_ZK_NODES=${KAFKA_ZK_SERVICE_HOST}:${KAFKA_ZK_SERVICE_PORT}, SPRING_CLOUD_STREAM_KAFKA_STREAMS_BINDER_BROKERS=${KAFKA_SERVICE_HOST}:${KAFKA_SERVICE_PORT},SPRING_CLOUD_STREAM_KAFKA_STREAMS_BINDER_ZK_NODES=${KAFKA_ZK_SERVICE_HOST}:${KAFKA_ZK_SERVICE_PORT},SPRING_REDIS_HOST=${REDIS_SERVICE_HOST},SPRING_REDIS_PORT=${REDIS_SERVICE_PORT}'
```

Comment:
If they are set as plain env-var's, can they not be set as individual k/v pairs in each line? Something to the effect of how the deployment YAML takes such k/v pairs for the Server. Here is an [example](https://docs.spring.io/spring-cloud-stream/docs/current/reference/htmlsingle/#_apache_kafka_streams_binder). It'd be easy on the eyes if it is possible to configure it that way.